zilencer/models.py

```diff
@@ -100,13 +100,16 @@ class RemoteRealmAuditLog(AbstractRealmAuditLog):
     """
 
     server = models.ForeignKey(RemoteZulipServer, on_delete=models.CASCADE)
-    realm_id = models.IntegerField(db_index=True)
+    realm_id = models.IntegerField()
     # The remote_id field lets us deduplicate data from the remote server
-    remote_id = models.IntegerField(db_index=True)
+    remote_id = models.IntegerField()
 
     def __str__(self) -> str:
         return f"{self.server!r} {self.event_type} {self.event_time} {self.id}"
 
+    class Meta:
+        unique_together = ("server", "remote_id")
```

Comment:
Do we want an index on `("server", "realm_id", "remote_id")` too, for queries that might iterate through all records for a given realm?

Reply:
I suppose, though we don't currently do any reading from this table.